Automattic/wp-calypso
# [nudge-a-palooza] Enable dark themes upsell nudges for everyone

Related to p9jf6J-Hl-p2

Updated nudges in `test` variation of themesNudgesUpdates won. This PR merges updated upsell nudges. There is less certainty about a nudge in the checkout process so we are moving it to it's own separate test.

Test plan:
1. As a free user, go to /themes/:site and confirm the premium upsell nudge is dark
1. Go to /themes/:theme/:site for a paid theme and confirm the premium upsell nudge is there and is dark
1. Click on theme preview and confirm the upsell nudge is there
